src/Numeric/Units/Dimensional/NonSI.hs

```diff
@@ -108,59 +118,296 @@ gee = mkUnitQ (ucumMetric "[g]" "g" "gee") 9.80665 $ meter / second ^ pos2
 Some US customary (that is, inch-pound) units.
 -}
 
-inch, foot, mil :: Fractional a => Unit 'NonMetric DLength a
-inch = mkUnitQ (ucum "[in_i]" "in" "inch") 2.54 $ centi meter
-foot = mkUnitQ (ucum "[ft_i]" "ft" "foot") 12 $ inch     -- 0.3048 m
-mil  = mkUnitQ (ucum "[mil_i]" "mil" "mil") 0.001 $ inch
-poundMass, ounce :: Fractional a => Unit 'NonMetric DMass a
-poundMass = mkUnitQ (ucum "[lb_av]" "lb" "pound") 0.45359237 $ kilo gram
-ounce     = mkUnitQ (ucum "[oz_av]" "oz" "ounce") (1 Prelude./ 16) $ poundMass
+-- | One international foot is one third of an international 'yard'.
```

Comment:
For this particular case, due to `3 *~ foot` defaulting to `0.9144000000000001 m` it would have to be something like:

    prop> 1 *~ foot == 1 *~ yard / _3

or, e.g., 

    >>> 3 *~ foot == (1 :: Rational) *~ yard
    True

or

    >>> import Data.AEq (AEq ((~==)))
    >>> 3 *~ foot ~== 1 *~ yard
    True

I think my preference, to the extent Rational is sufficient, is:

    -- $setup
    -- import Data.Ratio ((%))

    -- >>> 3 *~ foot == 1 % 1 *~ yard
    -- True

    -- >>> 1 *~ inch == 1 % 12 *~ foot
    -- True